### PR TITLE
[CS] Code Style fixes for administrator/components/com_templates/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style fixes

### Summary of Changes
- Expected 1 newline after opening brace
- Blank line found at start of control structure
- Line indented incorrectly
- Whitespace found at end of line
- Expected intention before asterisk
- Closing parenthesis of a multi-line function call must be on a line by itself
- Please consider an empty line before the return statement;
- No blank line found after control structure
- Tabs must be used to indent lines; spaces are not allowed

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

File: administrator/components/com_templates/controllers/template.php [hunk 1]
```diff
@@ -295,20 +295,20 @@ public function save()
...
-		case 'apply':
+			case 'apply':
 
-			// Redirect back to the edit screen.
-			$url = 'index.php?option=com_templates&view=template&id=' . $model->getState('extension.id') . '&file=' . $fileName;
-			$this->setRedirect(JRoute::_($url, false));
+				// Redirect back to the edit screen.
+				$url = 'index.php?option=com_templates&view=template&id=' . $model->getState('extension.id') . '&file=' . $fileName;
+				$this->setRedirect(JRoute::_($url, false));
 			break;
```
Comment (Quy) [inline administrator/components/com_templates/controllers/template.php]: Indent `break;`

File: administrator/components/com_templates/controllers/template.php [hunk 2]
```diff
@@ -295,20 +295,20 @@ public function save()
...
-			$file = base64_encode('home');
-			$id   = $app->input->get('id');
-			$url  = 'index.php?option=com_templates&view=template&id=' . $id . '&file=' . $file;
-			$this->setRedirect(JRoute::_($url, false));
+				// Redirect to the list screen.
+				$file = base64_encode('home');
+				$id   = $app->input->get('id');
+				$url  = 'index.php?option=com_templates&view=template&id=' . $id . '&file=' . $file;
+				$this->setRedirect(JRoute::_($url, false));
 			break;
```
Comment (Quy) [inline administrator/components/com_templates/controllers/template.php]: Indent `break;`

File: administrator/components/com_templates/controllers/template.php (hunk 1 above)
Comment (photodude) [reply administrator/components/com_templates/controllers/template.php]: Manual correction applied

File: administrator/components/com_templates/controllers/template.php (hunk 2 above)
Comment (photodude) [reply administrator/components/com_templates/controllers/template.php]: Manual correction applied

Comment (photodude) [reply administrator/components/com_templates/controllers/template.php]: (done does not like to play nice at times... RPC failed)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ede03dac0895988429d4bf9fa67b9cae348b111f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19081">issues.joomla.org/tracker/joomla-cms/19081</a>.</sub>

File: administrator/components/com_templates/controllers/style.php [hunk 3]
```diff
@@ -45,7 +45,6 @@ public function save($key = null, $urlVar = null)
 
 		if ($document->getType() == 'json')
```
Comment (C-Lodder) [inline administrator/components/com_templates/controllers/style.php]: `==` >> `===`

Comment (photodude) [reply administrator/components/com_templates/controllers/style.php]: Agreed. I generally don't make manual corrections in these PR's as the intent is to show and validate the [auto fixers from our PHPCS code style ruleset](https://github.com/joomla/coding-standards) which puts manual corrections beyond the scope of the PR. But I'll make an exception for this change.

**Comment by photodude** [comment]
Note: Drone failing is just Drone having problems. 
here is the unrelated drone error
```sh
error: RPC failed; curl 18 transfer closed with outstanding read data remaining 
fatal: The remote end hung up unexpectedly
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 95bedd247347aecf1e224dcec1cc9b02e5fd9490<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19081">issues.joomla.org/tracker/joomla-cms/19081</a>.</sub>

File: administrator/components/com_templates/controllers/template.php [hunk 4]
```diff
@@ -295,21 +295,21 @@ public function save()
 		// Redirect the user based on the chosen task.
 		switch ($task)
 		{
-		case 'apply':
+			case 'apply':
```
Comment (Quy) [inline administrator/components/com_templates/controllers/template.php]: Remove blank line. See example @ https://developer.joomla.org/coding-standards/php-code.html#user-content-a-switch-example

File: administrator/components/com_templates/controllers/template.php [hunk 5]
```diff
@@ -295,21 +295,21 @@ public function save()
...
-			$url = 'index.php?option=com_templates&view=template&id=' . $model->getState('extension.id') . '&file=' . $fileName;
-			$this->setRedirect(JRoute::_($url, false));
-			break;
+				// Redirect back to the edit screen.
+				$url = 'index.php?option=com_templates&view=template&id=' . $model->getState('extension.id') . '&file=' . $fileName;
+				$this->setRedirect(JRoute::_($url, false));
+				break;
 
-		default:
+			default:
```
Comment (Quy) [inline administrator/components/com_templates/controllers/template.php]: Remove blank line.

File: administrator/components/com_templates/controllers/template.php (hunk 4 above)
Comment (photodude) [reply administrator/components/com_templates/controllers/template.php]: I have removed the blank line proceeding the comment in the switch case

File: administrator/components/com_templates/controllers/template.php (hunk 5 above)
Comment (photodude) [reply administrator/components/com_templates/controllers/template.php]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 856a564fa7c56353ed512ac61035c207cd6eb049<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19081">issues.joomla.org/tracker/joomla-cms/19081</a>.</sub>

**Comment by photodude** [comment]
Thanks @wilsonge for the merges. This will put us just a little closer.

**Pull request merged**